jib-cli/src/main/java/com/google/cloud/tools/jib/cli/JibCli.java

```diff
@@ -33,6 +34,7 @@
 public class JibCli {
 
   static Logger configureHttpLogging(Level level) {
+    new ApacheHttpTransport(); // fixes https://github.com/GoogleContainerTools/jib/issues/3156
```

Comment:
Let's add a small comment to detail what this does? Something like, "to initialize a static logger field.."